site/docs/v1/tech/apis/hosted-backend.adoc

```diff
@@ -0,0 +1,396 @@
+---
+layout: doc
+title: Hosted Backend APIs
+description: APIs that you to perform OAuth 2 Authorization Code Flow login, logout, and refresh for applications
+---
+
+== Overview
```

Comment:
I tested this all with a simple HTML page with a handful of forms on it. It could be useful to include a plain HTML example of calling login, register, logout.

Reply:
I agree that would be useful but think it may be out of scope for the API documentation page. Would it be worth providing another fusionauth-example repo?